README.md

```diff
@@ -69,30 +69,30 @@ information to be available to users.
 
 #### `Repositories`
 
-`Repositories` describes the repositories that image-mirror interfaces with.
+`Repositories` describes the repositories that artifact-mirror interfaces with.
 This section roughly correlates to the `creds` section of `regsync.yaml`.
 
 | Field | Required | Description |
 | ------------- | ------------- |------------- |
-| `BaseUrl` | yes | The base URL for the repository. Appending `/` plus an image name should be a valid image reference.
+| `BaseUrl` | yes | The base URL for the repository. Appending `/` plus an artifact name should be a valid artifact reference.
 | `Password` | yes | The password to use when authenticating against the registry. See [the regsync documentation](https://regclient.org/usage/regsync/) for more details.
 | `Registry` | yes | The registry URL. See [the regsync documentation](https://regclient.org/usage/regsync/) for more details.
 | `ReqConcurrent` | no | The number of concurrent requests that are made to this registry. See [the regsync documentation](https://regclient.org/usage/regsync/) for more details.
-| `DefaultTarget` | no | Whether the Repository is used as a target repository for a given Image when the `TargetRepositories` field of the Image is not set.
+| `DefaultTarget` | no | Whether the Repository is used as a target repository for a given artifact when the `TargetRepositories` field of the artifact is not set.
```

Comment:
nit: these `artifact` occurrences could be capitalized, and maybe put in backticks, because they are references to the `Artifact` type.